src/themes/uclouvain/app/shared/object-collection/shared/badges/badges.component.html

```diff
@@ -1,9 +1,9 @@
 <ds-themed-status-badge [object]="object"></ds-themed-status-badge>
 <ng-container *ngIf="isMyDSpaceStatus">
     <ds-themed-my-dspace-status-badge [context]="context"></ds-themed-my-dspace-status-badge>
-    <span *ngIf="!isInWorkspace && context != Context.MyDSpaceArchived" class="text-light badge badge-workspace ">
+    <!-- <span *ngIf="!isInWorkspace && context != Context.MyDSpaceArchived" class="text-light badge badge-workspace ">
         {{ 'mydspace.status.submitted' | translate }}
-    </span>
+    </span> -->
```

Comment:
this is not correct : I want to know that my publication is "submitted" when it's not yet archived.
You need to remove this